docs/src/docusaurus-tree-sitter-plugin/index.js

```diff
@@ -0,0 +1,39 @@
+module.exports = function () {
+  return {
+    configureWebpack(config, isServer) {
+      let rules = [];
+
+      // Tree-sitter is only used for client-side code.
+      // Don't try to load it on the server.
+      if (isServer) {
+        rules.push({
+          test: /web-tree-sitter/,
+          loader: "null-loader",
+        });
+      } else {
+        // web-tree-sitter has a hard-coded path to tree-sitter.wasm,
+        // (see https://github.com/tree-sitter/tree-sitter/issues/559)
+        // which some browsers treat as absolute and others as relative.
+        // This breaks everything. Rewrite it to always use an absolute path.
+        rules.push({
+          test: /tree-sitter\.js$/,
```

Comment:
I couldn't figure out how to get webpack to match exactly node_modules/web-tree-sitter/tree-sitter.js. Anything more specific I used just made it not do anything.